docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,406 @@
+    "\n",
+    "- Eyeglasses (Person is wearing glasses)\n",
+    "- Wearing_Earrings (Person is wearing earings)\n",
+    "- Wearing_Hat (Person is wearing hat)\n",
+    "- Wearing_Necklace (Person is wearing a necklace)\n",
+    "- Wearing_Necktie (Person is wearing a necktie)\n",
+    "- No_Beard (beard is present)\n",
+    "- Smiling (Person is smiling)\n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "\n",
+    "- Find tokens with label issues using `cleanlab.filter.find_label_issues` with `multi_label = True`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "adaefc8b-b639-4bdf-af0d-337519e37ffc",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "We have `multi_labels` and `pred_probs` from a model trained on an existing data? Run the code below to get improved consensus labels and more information about the quality of your labels and annotators.\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```ipython3 \n",
+    "from cleanlab.filter import find_label_issues\n",
+    "\n",
+    "noise_idx = find_label_issues(\n",
+    "        labels=dataset[\"labels\"],\n",
+    "        pred_probs=dataset[\"pred_probs\"],\n",
+    "        multi_label=True,\n",
+    "        return_indices_ranked_by='normalized_margin',\n",
+    "    )\n",
+    "```\n",
+    "\n",
+    "    \n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a6261a3-6ea1-44a6-ac91-d375c8aa5535",
+   "metadata": {},
+   "source": [
+    "## 1. Install required dependencies and download data\n",
+    "\n",
+    "```ipython3\n",
+    "!pip install cleanlab\n",
+    "!pip install torchvision\n",
+    "\n",
+    "\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "\n",
+    "```\n",
+    "Kaggle Dataset source: [Kaggle CelebA](https://www.kaggle.com/datasets/jessicali9530/celeba-dataset)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7383d024-8273-4039-bccd-aab3020d331f",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# Package versions we used: matplotlib==3.5.1, torch==1.11.0, torchvision==0.12.0, timm==0.5.4\n",
+    "\n",
+    "dependencies = [ \"torchvision\", \"scikit-learn\"]\n",
```

Comment:
matplotlib, timm, torch should be in this list?

Reply:
I thought requirements.txt would cover, will add

Reply:
Just Matplotlib, this is the tutorial notebook, we don't need other libraries

Reply:
Added: https://github.com/cleanlab/cleanlab/pull/517/commits/d90d29e130901bbe1920df31d58a0102ccd2caf2